server/cmd/mmctl/commands/post.go

```diff
@@ -32,6 +32,14 @@ var PostCreateCmd = &cobra.Command{
 	RunE:    withClient(postCreateCmdF),
 }
 
+var PostDeleteCmd = &cobra.Command{
+	Use:     "delete [post_id]",
+	Short:   "Delete a post",
+	Example: `  post delete $POST_ID`,
```

Comment:
The example should not contain a placeholder. Please use an actual ID here so it's clear to the user what the command expects.